### ISSUE TITLE
Issue with a particular xpath only in TeamCity

### ISSUE THREAD

**Issue by P-Neha** [getgauge/taiko#1978]
Hi!

I'm having a weird issue where the xpath works in my local(Windows OS) and the same xpath fails when I try to run in the TeamCity(Linux) for CICD purpose. Here is the xpath I'm using - await $("//div[contains(@class,'TargetLocations')]//tbody//tr[1]/td[1]").text()

The xpath works in Mac OS and it works in the Taiko terminal on my Virtual Box - Ubuntu. Not sure what is happening. Could you please help me with this issue?

Thank You

**Comment by zabil** [comment]
> same xpath fails when I try to run in the TeamCity(Linux)

By fail do you mean it does not find the element? If so it might be because the application behaves differently on headless mode on team city. Have you tried taking a [screenshot](https://docs.taiko.dev/taking_screenshots/#default-screenshot) of the app on Teamcity before using the xpath?

**Comment by P-Neha** [comment]
Yes it's not able to find the element. I was able to capture the screenshot through the reports. So I observed there is a minor text wrap of the element I'm trying to click on. But I was wondering why the xpath would fail for that? From the screenshot it seems it was able to hover and click in that container but the clicked element doesn't do the needed action. Any help on this would be highly appreciate

**Comment by sriv** [comment]
Is there a way you could share a sample, that'd help us troubleshoot better

**Comment by P-Neha** [comment]
The website works only on company's vpn. Is it possible we could have a working session?
